rados/ioctx.go

```diff
@@ -220,7 +220,7 @@ func (ioctx *IOContext) GetPoolName() (name string, err error) {
 	for {
 		ret := C.rados_ioctx_get_pool_name(ioctx.ioctx,
 			(*C.char)(unsafe.Pointer(&buf[0])), C.unsigned(len(buf)))
-		if ret == -34 { // FIXME
+		if ret == -C.ERANGE {
```

Comment:
wow this is great. was there a time in the past when cgo didn't work with macro definitions?

Reply:
Not that I'm aware of, but I've only really been using cgo for about the last year and a half.